templates/cmd/controller/main.go.tpl

```diff
@@ -36,6 +36,13 @@ func main() {
 	flag.Parse()
 	ackCfg.SetupLogger()
 
+	if err := ackCfg.Validate(); err != nil {
+	    setupLog.Error(err, "Unable to create controller manager",
+	    "aws.service", awsServiceAlias,
+    	)
+    	os.Exit(1)
+    }
```

Comment:
@nithu0115 the indentation on the above is still off.  In your editor, make sure you replace four spaces with the TAB character and align the parentheses and brackets properly...